mars/dataframe/expressions/arithmetic/core.py

```diff
@@ -585,3 +587,226 @@ def __call__(self, x1, x2):
 
     def rcall(self, x1, x2):
         return self._call(x1, x2)
+
+class DataFrameUnaryOpMixin(DataFrameOperandMixin):
+    __slots__ = ()
+
+    @classmethod
+    def _tile_dataframe(cls, op):
+        df = op.outputs[0]
+        operand = op.inputs[0]
+
+        out_shape = operand.chunk_shape
+
+        out_chunks = []
+        for out_idx in itertools.product(*(range(s) for s in out_shape)):
+            # does not need shuffle
+            out_chunk = operand.cix[out_idx[0], out_idx[1]]
+            out_op = op.copy().reset_key()
+            out_chunks.append(
+                out_op.new_chunk([out_chunk], shape=(np.nan, np.nan),
+                                 index=out_idx))
+
+        # FIXME how `nsplits` works ?
+        nsplits = [[np.nan] * out_shape[0]] * out_shape[1]
+
+        new_op = op.copy()
+        return new_op.new_dataframes(op.inputs, df.shape, dtypes=df.dtypes,
+                                     index_value=df.index_value,
+                                     columns_value=df.columns,
+                                     chunks=out_chunks, nsplits=nsplits)
+
+    @classmethod
+    def tile(cls, op):
+        if isinstance(op.inputs[0], DATAFRAME_TYPE):
+            return cls._tile_dataframe(op)
+        raise NotImplementedError
+
+    @classproperty
+    def _operator(self):
+        raise NotImplementedError
+
+    @classmethod
+    def _calc_properties(cls, x1):
```

Comment:
For unary or constant op, we may not need this function any more, because the index, columns, or dtypes is identical to its input.